index.html

```diff
@@ -71,6 +71,250 @@ <h1 class="headline__title color-dark">
+							<img
+								src="img/work/port4.png"
+								alt="portfolio 4"
+								class="port-photo"
+							/>
+						</header>
+						<div class="port-body">
+							<h3 class="port-name color-dark">Multi-Post Stories</h3>
+							<div class="port-description flex-row align-center">
+								<h4 class="port-invent color-info">Canopy</h4>
+								<ul class="job_done flex-row">
+									<li class="items">Back End Dev</li>
+									<li class="items">2015</li>
+								</ul>
+							</div>
+							<p class="description-main">
+								A daily selection of privately personalized reads; no accounts
+								or sign-ups required.
+							</p>
+							<ul class="subjects flex-row align-center">
+								<li class="subject">HTML</li>
+								<li class="subject">CSS</li>
+								<li class="subject">JavaScript</li>
+							</ul>
+							<div class="click">
+								<a href="#" class="button__primary">See project</a>
+							</div>
+						</div>
+					</article>
+				</div>
+			</section>
+			<section
+				class="section section__white section__white--top"
+				id="about"
+			>
+				<div class="about flex-column justify-center">
+					<h2 class="about__title color-dark">About Myself</h2>
+					<p class="color-info">
+						Hello I'm a software developer! I can help you build a product,
+						feature or website. Look through some of my work and experience!
+						If you like what you see and have a project you need coded,
+						don't hesitate to contact me.
+					</p>
+					<a href="#">Let's Connect</a>
+					<ul class="social__icons flex-row align-center">
+						<li><a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a></li>
+						<li><a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a></li>
+						<li><a href=""><img src="img/m_icon.png" alt="Medium icon" /></a></li>
+						<li><a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a></li>
+						<li><a href=""><img src="img/finger_c.png" alt="finger icon" /></a></li>
+					
+					</ul>
+					</div>
+					<div class="about_page__actions">
+						<a href="#" class="button__primary"
+							>Get my resume</a
+						>
+					</div>
+					<div class="category-about_page">
+						<header
+							class="about_page__header flex-row justify-between align-center"
+						>
+							<h3 class="about_page__subtitle">Languages</h3>
+							<img src="img/about_page/arr_d.svg" alt="Arrow down icon" />
+						</header>
+						<ul class="about_page__list grid">
+							<li class="about__item">
+								<img
+									src="img/about_page/javascript.png"
+									alt="javascript png image"
+								/>
+								JavaScript
+							</li>
+							<li class="about_page__item">
+								<img
+									src="img/about_page/html.png"
+									alt="javascript png image"
+								/>
+								HTML
+							</li>
+							<li class="about_page__item">
+								<img
+									src="img/about_page/css.png"
+									alt="javascript png image"
+								/>
+								CSS
+							</li>
```

Comment:
Please make sure your design is the same as Figma design.
Your design

![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/101523582/5bad4e69-0924-47e4-be0a-238a58be320e)
 Figma design

![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/101523582/a6300e87-d7ff-4fdc-939d-7c943737cf68)